src/encoded/static/libs/bootstrap/pager.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
why is this in libs/bootstrap/?  (*looks at code*)  I guess the salient question is - why do we have bootstrap as a part of libs/ anyway?  Is is the set of components that bootstrap-scss applies to?